sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSparkSubmitSuite.scala

```diff
@@ -129,6 +129,19 @@ class HiveSparkSubmitSuite
     runSparkSubmit(args)
   }
 
+  test("SPARK-12662 fix DataFrame.randomSplit to avoid creating overlapping splits") {
```

Comment:
Do we need to start a whole new process to test this? I think we can just run randomSplit in the normal DataFrameSuite?